package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "lint": "eslint **/*.js",
     "test": "jest",
+    "default-tests": "jest --selectProjects default-tests",
+    "serial-tests": "jest --selectProjects serial-tests",
```

Comment:
Since you are modifying npm scripts. In my opinion serialisation could be introduced with additional match pattern and passing parameter runInBand. In this case it would not need additional dependency (jest-serial-runner).

Reply:
Will try this approach.